deployment/terraform/resources/keyvault.tf

```diff
@@ -66,12 +66,14 @@ data "azurerm_key_vault_secret" "backend_app_id" {
 
 ## Access Policies
 
+# It's unclear who should own this vault access policy.
```

Comment:
It's a bit unfortunate, but I think this needs to be managed manually. We can't have both the `dev` and `staging` stacks managing this access policy.